os/binfmt/libelf/libelf_init.c

```diff
@@ -95,6 +95,52 @@
  * Private Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: bin_header_info
+ *
+ * Description:
+ *  Get the binary header info
+ *
+ * Returned Value:
+ *   0 (OK) is returned on success and a negated errno is returned on
+ *   failure.
+ *
+ ****************************************************************************/
+ #ifdef CONFIG_ELF_MTD
+ struct bininfo {
+	 int st_size;
+	 int hdr_size;
+ }bininfo;
```

Comment:
Please add space after '{'

Reply:
Corrected, dummy structure will remove when binary manager defines the actual structure